From e8ff1e7e5d3b8613c9d8d18b7cebb9641d64e2e3 Mon Sep 17 00:00:00 2001
From: Siddharth Seth <sseth@apache.org>
Date: Tue, 18 Oct 2016 16:05:41 -0700
Subject: [PATCH 1019/1164] CDH-48309 HIVE-14973. Fix Flaky test:
 TestJdbcWithSQLAuthorization.testBlackListedUdfUsage.
 (Zoltan Haindrich, reviewed by Siddharth Seth)

(cherry picked from commit 57044c47fb4b9cff2e3e3cea1b3fcf5d00df11a4)

Change-Id: I042e486f4ef84b4dfe830b2922c5ec7b27b50c56
---
 .../TestJdbcWithSQLAuthUDFBlacklist.java           |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/itests/hive-unit/src/test/java/org/apache/hive/jdbc/authorization/TestJdbcWithSQLAuthUDFBlacklist.java b/itests/hive-unit/src/test/java/org/apache/hive/jdbc/authorization/TestJdbcWithSQLAuthUDFBlacklist.java
index b111f95..5070c76 100644
--- a/itests/hive-unit/src/test/java/org/apache/hive/jdbc/authorization/TestJdbcWithSQLAuthUDFBlacklist.java
+++ b/itests/hive-unit/src/test/java/org/apache/hive/jdbc/authorization/TestJdbcWithSQLAuthUDFBlacklist.java
@@ -72,7 +72,7 @@ public void testBlackListedUdfUsage() throws Exception {
     Connection hs2Conn = DriverManager.getConnection(miniHS2.getJdbcURL(), "user1", "bar");
 
     Statement stmt = hs2Conn.createStatement();
-    String tableName1 = "test_jdbc_sql_auth_udf";
+    String tableName1 = "test_jdbc_sql_auth_udf_blacklist";
     stmt.execute("create table " + tableName1 + "(i int) ");
 
     verifyUDFNotAllowed(stmt, tableName1, "sqrt(1)", "sqrt");
-- 
1.7.9.5

